vlsi/example-vlsi-nangate45

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+import os
+
+import hammer_vlsi
+from hammer_vlsi import CLIDriver, HammerToolHookAction
+
+from typing import Dict, Callable, Optional, List
+
+def example_place_tap_cells(x: hammer_vlsi.HammerTool) -> bool:
+    x.append("")
+    return True
+
+def example_add_fillers(x: hammer_vlsi.HammerTool) -> bool:
+    x.append("")
+    return True
+
+class ExampleDriver(CLIDriver):
+    def get_extra_par_hooks(self) -> List[HammerToolHookAction]:
```

Comment:
I think we just have a generic exampleDriver that inspects `os.env(tech_name)` and includes the get hooks programmatically.

The magic of hammer 🎉